philology/src/test/java/com/jcminarro/philology/transformer/TextViewTransformerTest.kt

```diff
@@ -61,13 +73,59 @@ class TextViewTransformerTest {
 
     @Test
     fun `Should reword text and hint`() {
-        val viewResult = TextViewTransformer.reword(textView, createAttributeSet(
+        val viewResult = TextViewTransformer.reword(
+            textView, createAttributeSet(
                 ResourceIdAttribute("text"),
-                ResourceIdAttribute("hint")))
+                ResourceIdAttribute("hint")
+            )
+        )
 
         viewResult `should be` textView
         Verify on textView that textView.setText(0) was called
         Verify on textView that textView.setHint(1) was called
         `Verify no further interactions` on textView
     }
-}
\ No newline at end of file
+
+    @Test
+    fun `Should reword only text from style`() {
```

Comment:
Same here about xml-attribute vs style-attribute

Reply:
Covered this use-case in a separate unit test.